### ISSUE TITLE
Get an  empty polygon than can be cut, but empty inside?

### ISSUE THREAD

**Issue by ThomasCartier** [mthh/sfcgal-rs#5]
Hi,

Not sure if you will be able to answer this but I have a boolean operation occuring on a polygon and the result is the same as the figure on the [right](https://postgis.net/docs/manual-dev/CG_3DDifference.html):
the cut polygon has its "walls" redefined inside.

Instead it should not have walls as my image below were the cross show the walls that should not appear:
![Screenshot_20240901_175427](https://github.com/user-attachments/assets/51960796-6ee2-4e7d-bb05-d00af0330e13)

In other words, the polygon should behave like a surface.

How to do it ?

(The MultiPoint type won't make it : it won't give full triangulation for 3d reconstruction.)

**Comment by ThomasCartier** [comment]
To answer my own question, yes, just create the polygon with an interior!
